server/controllers/work.js

```diff
@@ -31,6 +32,35 @@ function getImageIndex(ctx) {
   return imageIndex;
 }
 
+function constructCreatorsString(creators) {
+  if (creators.length > 0) {
+    const creatorsString =  creators.reduce((acc, creator, index) => {
+      if (index === 0) {
+        return `${acc} ${creator.label}`;
+      } else if (index + 1 === creators.length) {
+        return `${acc} and ${creator.label}`;
+      } else {
+        return `${acc}, ${creator.label}`;
+      }
+    }, 'by');
```

Comment:
```JS

creators.map(creator => creator.label).join(', ').replace(/,(?!.*,)/gmi, ', and');

```?

Reply:
Nice, but won't work correctly as some of the creator.label text contains commas, e.g. 'Gillray, James', which would end up as Gillray, and James